.github/workflows/configs/.markdownlint.json

```diff
@@ -0,0 +1,3 @@
+{
+    "MD013": false
+}
```

Comment:
Add a new line at the end of the file.

Reply:
That's my JSON auto-formatter, it does remove trailing new lines for json files 😄